app/meldingen/incident/components/IncidentPreview/components/PlainText/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Text = ({ text }) => (
```

Comment:
Is this the style for the props we're using throughout the entire app? Or do we want to pass it as `(props)` and use it as `props.text`? I have no preference, but I think we should be sure that one style is used